### DISCUSSION TITLE
Can I use the MemoryStream Class ?

### DISCUSSION THREAD

**Discussion by spechtnathan** [SebLague/Chess-Challenge#160] [Q&A] [answered]
Can I use the [MemoryStream Class](https://learn.microsoft.com/en-us/dotnet/api/system.io.memorystream?view=net-7.0) ?

**Comment by SebLague** [comment, marked as answer]
Hi, I'm afraid you can't use anything inside the System.IO namespace

**Reply by vivek09thakur** [reply]
Hi, I'm afraid you can't use anything inside the System.IO namespace

**Comment by vivek09thakur** [comment]
Hi, I'm afraid you can't use anything inside the System.IO namespace

**Comment by vivek09thakur** [comment]
Hi, I'm afraid you can't use anything inside the System.IO namespace
